packages/shylu/shylu_node/tacho/src/impl/Tacho_NumericTools_LevelSet.hpp

```diff
@@ -2491,7 +2503,7 @@ class NumericToolsLevelSet : public NumericToolsBase<ValueType, DeviceType> {
       auto vecY = ((nlvls-1-lvl)%2 == 0 ? vecW : vecL);
       if (s0.spmv_explicit_transpose) {
         status =
-          #if ROCM_VERSION >= 50400
+          #if (ROCM_VERSION >= 50400 && ROCM_VERSION < 60000)
```

Comment:
Looks like this could be made more consistent with similar changes by swapping the order of the check and function api, e.g. change to:
```
      #if (ROCM_VERSION >= 60000)
      rocsparse_spmv
      #else
      rocsparse_spmv_ex
```